app/views/SidebarView/index.js

```diff
@@ -60,6 +60,10 @@ class Sidebar extends Component {
 		};
 	}
 
+	componentWillMount() {
```

Comment:
Hello @Ritesh-Ag , I think this method isn't the best approach to resolve this issue, `componentWillMount` [will be deprecated](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount) soon.
I think this problem is related to database asynchronous, we try to read permissions from database before this was insert.

Reply:
@djorkaeffalexandre Okay Thank you so much sir. Can you please suggest me what should be the best approach to solve the issue?